decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb

```diff
@@ -1039,9 +1039,46 @@
 
       it "shows a message indicating that comments are disabled" do
         visit resource_path
-        expect(page).to have_content("Comments are disabled at this time")
+        expect(page).to have_content("Comments are currently disabled, only administrators can reply or post new ones.")
         expect(page).to have_no_content("You need to be verified to comment at this moment")
       end
+
+      context "when comments are blocked and the user is an administrator" do
+        let!(:user) { create(:user, :admin, :confirmed, organization:) }
+
+        before do
+          login_as user, scope: :user
+          visit resource_path
+        end
```

Comment:
- no need for this, already in lines 1028-1031

Reply:
Done